facebook.js

```diff
@@ -78,10 +78,11 @@ const emoji_map = {
   "wedding": "💒"
 }
 
-const show_facebook_cv_tags = function() {
+const show_facebook_cv_tags = function(obj) {
   const TAG_PREFIX = "Image may contain: ";
-  const images = [...document.getElementsByTagName('img')];
-
+  let images = [...obj.querySelectorAll('img.img')];
+  images = images.concat([...obj.querySelectorAll('img.spotlight')]);
+  // console.log(images);
```

Comment:
Probably want to remove the comment :P
This looks great though (I'm not a collaborator)